docs/index.rst

```diff
@@ -73,28 +74,37 @@ The guide is split up into two sections:
       
        Technical information - specifications, APIs, architecture
 
-Having trouble? We would like to help!
-======================================
+Project and community
+---------------------
 
-If you run into problems: don't panic! Check out the :ref:`communication
-article<communication>` and you will find out how to most easily get in
-touch with other developers.
+The Ubuntu Packaging Guide is an open source project that warmly welcomes
+community projects, contributions, suggestions, fixes and constructive feedback. Read our `Code of Conduct`_ to get started.
 
-- Links to other communication channels go here
-- Use the "Give feedback" button at the top of any page to open a GitHub issue
-  and let us know what problem you're having
+If you run into problems, don't panic! The following communication channels are there to help you through:
 
-Project and community
-=====================
+Mailing lists:
+~~~~~~~~~~~~~~
+In a project where thousands of lines of code are changed, lots of decisions
+are made and hundreds of people interact every day, it is important to 
+communicate effectively.
 
-The Ubuntu Packaging Guide is an open source project that warmly welcomes
-community projects, contributions, suggestions, fixes and constructive feedback.
+* https://lists.ubuntu.com/mailman/listinfo/ubuntu-devel-announce 
+  (announce-only, the most important development announcements go here)
+* https://lists.ubuntu.com/mailman/listinfo/ubuntu-devel
+  (general Ubuntu development discussion)
+* https://lists.ubuntu.com/mailman/listinfo/ubuntu-motu
+  (MOTU Team discussion, get help with packaging)
+
+IRC channels:
+~~~~~~~~~~~~~
+For real-time discussions, please connect to irc.libera.chat and join one or
+any of these channels:
 
-* Read our `Code of Conduct`_
-* IRC?
-* Discourse?
-* Mailing list?
-* GitHub?
+* #ubuntu-devel (for general development discussion)
+* #ubuntu-motu (for MOTU team discussion and generally getting help)
```

Comment:
These channels do still exist and are still used. I'm not sure if there are any other channels that should be included - or any that the Community team would like us to add. Could be worth checking with Aaron?

Reply:
*thought:* The problem is that it depends on what you want to do in Ubuntu. Recommending those two in general seems fine to me.

---

*issue:* We have to explain to beginners what "MOTU" (Masters of the Universe, see https://wiki.ubuntu.com/MOTU) is. 

*thought:* I thought of adding a glossary (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/glossary.html) for terms like MOTO, FTBFS, NBS, ... that show up frequently in Ubuntu communication. Maybe we should even add a small glossary for slang abbreviations like ACK, IIRC, AFAICT, ... Because they confused the heck out of me when I started.

Reply:
That's a great idea!

Reply:
Glossary is a good idea. For now, I have included the link and the expanded form of MOTU.